api/all/build.gradle.kts

```diff
@@ -23,3 +26,23 @@ tasks.test {
   // Configure environment variable for ConfigUtilTest
   environment("CONFIG_KEY", "environment")
 }
+
+graalvmNative {
+  binaries.all {
+    resources.autodetect()
+
+    // Required as of junit 5.10.0: https://junit.org/junit5/docs/5.10.0/release-notes/#deprecations-and-breaking-changes
+    buildArgs.add("--initialize-at-build-time=org.junit.platform.launcher.core.LauncherConfig")
+    buildArgs.add("--initialize-at-build-time=org.junit.jupiter.engine.config.InstantiatingConfigurationParameterConverter")
+
+    // Required by JUnit 4
+    buildArgs.add("--initialize-at-build-time=org.junit.validator.PublicClassValidator")
+  }
+
+  // org.graalvm.buildtools.native pluging requires java 11+ as of version 0.9.26
+  tasks {
+    withType<JavaCompile>().configureEach {
+      options.release.set(11)
```

Comment:
Well this is your problem @jeanbisutti. We publish components with java 8 compatibility. This tries to overwrite that, and then other components which depend on this module fail because they specify java 8 compatibility.

Maybe you want to change the version to java 11 for test tasks? But even that might be too wide a net to cast, since I don't think we want java 11 syntax in test source and java 8 in main source.